addons/web_editor/models/ir_ui_view.py

```diff
@@ -172,7 +172,8 @@ def _copy_field_terms_translations(self, record_from, name_field_from, record_to
             record_from[name_field_from],
             {lang: record_from.with_context(prefetch_langs=True, lang=lang)[name_field_from] for lang in langs if lang != lang_env}
         )
-        existing_translation_dictionary.update(extra_translation_dictionary)
+        if not any(translated_value != key for key in extra_translation_dictionary.keys() for translated_value in existing_translation_dictionary[key].values()): 
+            existing_translation_dictionary.update(extra_translation_dictionary)
```

Comment:
1. the method name is `_copy_field_terms_translations` which is a very simple and intuitive method. We should try not changing its feature by introducing this extra `not copy` logic. (after this fix, the definition becomes 'copy translations from record_from to record_to if all these new translations are all not translated in record_to). If it is necessary, we should also rename the method to avoid anyone else abusively using it.
2. not sure if it is acceptable. we could use an extra parameter `overwrite=False` which will keep customized exsiting translations by using comparing if a term is translated or not.